backend/src/main/java/com/jobwait/persistence/answerpersistence/LocalDateAnswer.java

```diff
@@ -41,8 +40,8 @@ public void setSQLStatement(PreparedStatement statement, Integer statementIndex)
         if (this.getValue() == null) {
             statement.setNull(statementIndex, this.sqlType);
         } else {
-            statement.setTimestamp(statementIndex,
-                    Timestamp.from((this.getValue()).toInstant()));
+            statement.setObject(statementIndex,
```

Comment:
I don't trust `setObject`, please use `setDate`. https://stackoverflow.com/a/18615191

Reply:
I skimmed through it, I didn't see any reason to not use `setDate` besides the fact that `setObject` works. I'd rather use `setDate` simply because java sql implements it and so why not use it (maybe they've thought of something we haven't, plus it's more readable). Is there a specific problem with `setDate` not working?

Reply:
the answer is right there, so that you dont have to read it, i read it for you. setDate only takes objects of type java.sql.Date which is outdated and eclipsed by java.time objects, which LocalDate is a part of. In order to continue not using outdated classes we use setObject instead.

Reply:
gotcha. I'm surprised the SQL library uses an outdated java concept, but in that case using setObject makes sense